react/SquareAppIcon/index.jsx

```diff
@@ -119,7 +119,7 @@ export const SquareAppIcon = ({ app, name, variant, IconContent }) => {
               ) : IconContent ? (
                 IconContent
               ) : (
-                <AppIcon app={app} />
+                <AppIcon app={app} type={type} />
```

Comment:
I'm not a big fan of adding new prop like this. We can now set the `type` but what about other AppIcon's props? https://github.com/cozy/cozy-ui/blob/master/react/AppIcon/index.jsx#L136-L149 . I'm confident that you open a PR today to pass the `priority` props to SquareAppIcon. And we should not do that. 

I'm wondering if we should not pass an `appIconProps` to SquareAppIcon to be able to pass the props we want to 

cc @doubleface @JF-Cozy

Reply:
looks like appIconProps is a good idea